src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,70 @@
+.stopwatch {
+  position: relative;
+  width: $width;
+  height: $width;
+  border: $border-width dotted $border-color;
+  border-radius: 50%;
+
+  &__minutes-hand {
+    position: absolute;
+    height: $minutes-hand-height;
+    width: $hand-width-minutes;
+    background-color: $minutes-hand-color;
+    bottom: 50%;
+    left: calc(50% - #{$hand-width-minutes} / 2);
```

Comment:
It seems there is a mistake in the calculation of the left property for the `.stopwatch__minutes-hand`. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), calculations should be consistent. Here, `calc(50% - #{$hand-width-minutes} / 2)` uses a Sass variable inside the `calc` function, but the division is not wrapped in parentheses which might lead to unexpected results. It should be `calc(50% - (#{$hand-width-minutes} / 2))`.